### PR TITLE
added support for Geonames

### PR DESCRIPTION
**Pull request by stevro**

### CODE REVIEW

File: DependencyInjection/BazingaGeocoderExtension.php [hunk 1]
```diff
@@ -247,6 +247,14 @@ public function load(array $configs, ContainerBuilder $container)
                 }
             }
         }
+
+        if (isset($config['providers']['geonames'])) {
+            $ipInfoDbParams = $config['providers']['geonames'];
```
Comment (toin0u) [inline DependencyInjection/BazingaGeocoderExtension.php]: I think `$geonamesParams` is better here ;) Same in the line 255.

**Comment by sivolobov** [comment]
Why is it not merged yet?
Travis errors aren't related to the PR itself.

**Comment by Nyholm** [comment]
Could you rebase your PR?

**Comment by Nyholm** [comment]
I've changed the base of this PR. If you are interested to get it merged, you have to rebase to 4.x branch.

**Comment by Nyholm** [comment]
Closing this because of inactivity

**Pull request closed without merging**